### PR TITLE
Implemented GUID for patient ID.  

### PR DESCRIPTION
**Pull request by jeremyford-ucsd**
Full GUID is too long for daily.co name - used first 20 characters and removed 'patient-' prefix to make shorter.

### CODE REVIEW

File: frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs [hunk 1]
```diff
@@ -29,8 +29,7 @@ public PocPatientRepository(IPhysicianRepository physicianRepository)
 
         public Task<string> AdmitPatient(Patient patient)
         {
-            _count++;
-            patient.Id = $"{_count}";
+            patient.Id = Guid.NewGuid().ToString("N").Substring(0,20);
```
Comment (stephen-barrow) [inline frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs:32]: what i found is that if you replace the hyphens it works
ie  Guid.NewGuid().ToString().Replace("-",string.empty);
just on the off chance there is identical guids at that length

File: frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ public interface IPatientRepository
     public class PocPatientRepository : IPatientRepository
     {
         private static List<Patient> _admittedPatients = new List<Patient>();
-        private static int _count;
+        //private static int _count;
```
Comment (stephen-barrow) [inline frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs:22]: delete

File: frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/patient-detail/patient-detail.component.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ export class PatientDetailComponent implements OnInit {
 
   constructor(route: ActivatedRoute) {
     route.params.subscribe((p) => {
-      this.room = `patient-${p['id']}`;
+      this.room = p['id]'];
```
Comment (stephen-barrow) [inline frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/patient-detail/patient-detail.component.ts]:
```suggestion
      this.room = p['id];
```

Comment (jeremyford-ucsd) [reply frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/patient-detail/patient-detail.component.ts]: This was fixed in a previous commit

File: frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs (hunk 2 above)
Comment (jeremyford-ucsd) [reply frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs:22]: Deleted

File: frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs (hunk 1 above)
Comment (jeremyford-ucsd) [reply frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs:32]: I think the ToString("N") method removes the hyphens.
Removed .Substring()

**Pull request merged**